tests/regression_tests/mgxs_library_condense/results_true.dat

```diff
@@ -214,16 +214,16 @@
 28      2  2  y-min out        1   total  4.548  0.156691
   mesh 1       group in nuclide      mean std. dev.
        x  y  z                                     
-0      1  1  1        1   total  0.757948  0.035459
-2      1  2  1        1   total  0.779112  0.047034
-1      2  1  1        1   total  0.787475  0.050368
-3      2  2  1        1   total  0.769656  0.058065
+0      1  1  1        1   total  0.931945  0.045510
```

Comment:
I presume you verified these new values with hand-calculations of the pre-condensed results?